### ISSUE TITLE
Let out('smoo') be spelled as just 'smoo'

### ISSUE THREAD

**Issue by erikrose** [mozilla/fathom#51]
…for symmetry with facts.get('smoo').

**Comment by danielhertenstein** [comment]
Can you provide more detail. I've looked in the repo for `out('smoo')` and `facts.get('smoo')`, but don't see either anywhere.

**Comment by erikrose** [comment]
Yes, sorry. "smoo" is just a placeholder, like "foo" or "bar". Translating from "Erik-ese", the intent of this ticket is to let rules like…

```
rule(type('foo').max(), out('bar'))
```

…be more concisely represented as…

```
rule(type('foo').max(), 'bar')
```

It should be a matter of "compiling" a RHS string down to `out(theString)` during ruleset or rule construction.

**Comment by danielhertenstein** [comment]
Do we want to allow for rules to still be written the previous way? I think we'll have to so people can still do things like: https://github.com/mozilla/fathom/blob/5cf2e902d3cdd01368ed72e7796b848f741e17b0/test/rhs_tests.mjs#L86

**Comment by erikrose** [comment]
Yep!

**Comment by danielhertenstein** [comment]
Closed by #124
